src/Microsoft.AspNetCore.Razor.Tasks/DotnetToolTask.cs

```diff
@@ -62,12 +76,19 @@ public override bool Execute()
                 }
             }
 
-            if (TryExecuteOnServer(out var result))
+            return base.Execute();
+        }
+
+        protected override int ExecuteTool(string pathToTool, string responseFileCommands, string commandLineCommands)
```

Comment:
I assumed MsBuild takes care of providing errors when a required argument or option is missing. Otherwise we just want to pass it through to the compiler/tool.

Reply:
![image](https://user-images.githubusercontent.com/1430011/35066662-44e63a70-fb86-11e7-8a29-a937ce0948ac.png)